test/integration/agent/openflow_test.go

```diff
@@ -59,14 +59,13 @@ var (
 )
 
 const (
-	ingressRuleTable    = uint8(90)
-	ingressDefaultTable = uint8(100)
-	contrackCommitTable = uint8(105)
+	ingressRuleTable    = "IngressRule"
+	ingressDefaultTable = "IngressDefaultRule"
 	priorityNormal      = 200
 )
 
 type expectTableFlows struct {
-	tableID uint8
+	tableID string
```

Comment:
Why not rename it to tableName? In my opinion, this may cause some misunderstanding(In fact, I have confused before seeing this change).

Reply:
Updated.